lighthouse-viewer/test/viewer-test-pptr.js

```diff
@@ -59,80 +65,173 @@ describe('Lighthouse Viewer', function() {
     });
     viewerPage = await browser.newPage();
     viewerPage.on('pageerror', pageError => pageErrors.push(pageError));
-    await viewerPage.goto(viewerUrl, {waitUntil: 'networkidle2', timeout: 30000});
-    const fileInput = await viewerPage.$('#hidden-file-input');
-    await fileInput.uploadFile(sampleLhr);
-    await viewerPage.waitForSelector('.lh-container', {timeout: 30000});
   });
 
   afterAll(async function() {
     // Log any page load errors encountered in case before() failed.
     // eslint-disable-next-line no-console
-    console.error(pageErrors);
+    if (pageErrors.length > 0) console.error(pageErrors);
 
     await Promise.all([
       new Promise(resolve => server.close(resolve)),
       browser && browser.close(),
     ]);
   });
 
+  describe('Upload', () => {
```

Comment:
This is mostly whitespace changes. Moved the existing tests into it's own `Upload` describe, and made a new one for loading from PSI.